gradle.properties

```diff
@@ -3,7 +3,7 @@
 pluginGroup=com.sourcegraph.jetbrains
 pluginName=Sourcegraph
 # SemVer format -> https://semver.org
-pluginVersion=6.0-localbuild
```

Comment:
It was annoying that `runIde` runs always showed update available notification. This is the fix for that.